### PR TITLE
Deployment using docker-compose and doc associated

### PR DESCRIPTION
**Pull request by iksopo**
This deployment uses docker-compose and requires a Dockerfile to set up the container (tested using #4 PR).
In this PR I added deployment using docker-compose, how to set it up in README.md and the corresponding docker-compose.yml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,4 +14,26 @@ Some ideas for obtaining a :gift: if you are the first that:
 User name | NIA | Travis-CI|Score
 ----------|-----|----------|-----
 [UNIZAR-30246-WebEngineering](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race) |30246 | [![Build Status](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/actions/workflows/ci.yml/badge.svg)](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/actions/workflows/ci.yml)
-your name | your nia | your Travis-ci status
+[Óscar Pueyo Ciutad](https://github.com/iksopo) | 780378 | [![Build Status](https://github.com/iksopo/lab1-git-race/actions/workflows/ci.yml/badge.svg)](https://github.com/iksopo/lab1-git-race/actions/workflows/ci.yml)
+
```
Comment (fjlopez) [inline README.md:18]: Move to *description.md*

**Comment by fjlopez** [comment]
Some changes requested. Don't close this PR.

**Comment by iksopo** [comment]
Changes done.

**Comment by iksopo** [comment]
More conflicts solved

File: description.md [hunk 2]
```diff
@@ -91,7 +91,29 @@ The file `src/test/kotlin/IntegrationTest.kt` contains two tests that checks the
 
 This ensures the webpage has a valid CSS file.
 
-## How to deploy the app with Heroku
+## Deployment
```
Comment (fjlopez) [inline description.md:94]: Markdown file linted and fixed. I recommend you to use tools such as [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)

**Pull request merged**